packages/auth/src/platform_browser/recaptcha/recaptcha_loader.ts

```diff
@@ -77,7 +77,7 @@ export class ReCaptchaLoaderImpl implements ReCaptchaLoader {
           return;
         }
 
-        // Wrap the greptcha render function so that we know if the developer has
+        // Wrap the recaptcha render function so that we know if the developer has
```

Comment:
It's possible this should be `grecaptcha`, but based on the code below using `recaptcha`, I guessed this way...